ui-components.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+    "folders": [
+        {
+            "path": "."
+        }
+    ]
+}
```

Comment:
Probably this file should not be in version control.  I'd say either add it to the `.gitignore` or you could add it to your global `.gitignore` (https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)